libr/bin/pdb/pdb.c

```diff
@@ -485,6 +485,7 @@ static bool pdb7_parse(RPdb *pdb) {
 	}
 	p_tmp = root_page_data;
 	for (i = 0; i < num_root_index_pages; i++) {
+		if (UT64_MUL_OVFCHK(root_index_pages[i], page_size)) { break; }
```

Comment:
Follow the coding style guidelines:
* missing space before (
* newlines after { and ;

Reply:
One problem with the usage of UT64_MUL_OVFCHK is that we still have the lgtm warning (Multiplication result converted to larger type).

Reply:
We could maybe use some comment to discard the warning.

Reply:
https://lgtm.com/help/lgtm/alert-suppression#c-c++